nrf_modem/doc/softsim_interface.rst

```diff
@@ -137,10 +139,7 @@ The following code block shows an example implementation of the API. The `softsi
    }
 
 .. note::
-
    The :c:type:`nrf_modem_softsim_req_handler_t` handler is called in an interrupt context so it is recommended to handle the requests asynchronously.
 
-.. note::
-
-   The :c:member:`NRF_MODEM_SOFTSIM_RESET` request is issued whenever the processing of a request becomes unresponsive.
-   The :c:member:`NRF_MODEM_SOFTSIM_RESET` request shall thus be handled in an independent thread context with higher priority than the other commands to guarantee responsiveness of the application.
+   The :c:enumerator:`NRF_MODEM_SOFTSIM_RESET` request is issued whenever the processing of a request becomes unresponsive.
+   The :c:enumerator:`NRF_MODEM_SOFTSIM_RESET` request will thus be handled in an independent thread context with higher priority than the other commands to guarantee responsiveness of the application.
```

Comment:
This is up to the application. Thus, I think `will` is not the correct term here.